modal/utility/utility-icon-size.css

```diff
@@ -0,0 +1,17 @@
+  /*---------- icon-size*/
+
+  .icon-size-large{
+    font-size: 5rem !important;
+  }
+  
+  .icon-size-mediun{
+    font-size: 2rem !important;
+  }
+  
+  .icon-size-small{
+    font-size: 1rem !important;
+  }
+  
+  .icon-size-extrasmall{
+    font-size: 0.5rem !important;
+  }
```

Comment:
avoid using important

Reply:
I can't change the size of an icon (material-ui) without  using - "!important" .